app/src/components/Incident.js

```diff
@@ -5,15 +5,36 @@ import * as apiClient from "../apiClient";
 
 import IncidentMap from "./IncidentMap";
 import IncidentSummary from "./IncidentSummary";
+import Weather from "./Weather";
 const Incident = ({ incidentNumber }) => {
   const [incident, setIncident] = React.useState([]);
-
+  const [lat, setLat] = React.useState([]);
+  const [long, setLong] = React.useState([]);
+  const [data, setData] = React.useState([]);
```

Comment:
I have a personal rule that `data` is never a good name for a variable. Can you use something more descriptive?